carrierwave-activerecord.gemspec

```diff
@@ -41,13 +41,13 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
 
   # CarrierWave has broken in 0.x releases.
-  gem.add_runtime_dependency 'carrierwave', '~> 0.8.0'
+  gem.add_runtime_dependency 'carrierwave', '~> 1.0'
```

Comment:
this was preventing intoxitrack service's gemfile from using the new carrierwave version